guardian/testapp/migrations/0010_auto_20170628_1735.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```

Comment:
Django always creates that migration. You can run it over and over and it will keep creating it. Should I just remove it from this PR?